### ISSUE TITLE
[Icon Request] AfterPay

### ISSUE THREAD

**Issue by michaelvermeulen** [AlexanderPoellmann/PaymentFont#147]
Hi!

I'd like to see "AfterPay" as a payment method in the awesome payment-font library.

**Comment by michaelvermeulen** [comment]
Any news about this specific icon?

**Comment by AlexanderPoellmann** [comment]
Yep, it will be included in the next release. I don't have a release date though as I am quite busy at work at the moment.
